test/actions/pulp3/orchestration/remove_orphans_test.rb

```diff
@@ -55,7 +55,7 @@ def test_orphans_are_removed
       repository_reference = repo_reference(@repo)
       versions = ::Katello::Pulp3::Api::File.new(@primary).repository_versions_api.list(repository_reference.repository_href, {}).results.collect(&:pulp_href)
       tasks = ::Katello::Pulp3::Api::File.new(@primary).tasks_api.list(state__in: ['completed']).results
-      assert_empty tasks
```

Comment:
We check the that the tasks are empty here because we purge tasks in Pulp as part of the orphan cleanup process

Reply:
Ah ok, now I get it.

But the `purge`-task does(/cannot) purge itself, does it?
So we should exclude it from the results in `tasks`.

I have modified my change accordingly.

Reply:
I've seen confusing behavior around this in the past. Unfortunately the best way I found to have the test record properly was to completely reset the box and re-run the test.

It also sounds odd that the purge task would purge itself, but I suppose that was the case at some point. Perhaps it changed.